docs/reference/NuGet-Client-SDK.md

```diff
@@ -10,9 +10,30 @@ ms.topic: conceptual
 # NuGet Client SDK
 
 The *NuGet Client SDK* refers to a group of NuGet packages:
-
-* [`NuGet.Protocol`](https://www.nuget.org/packages/NuGet.Protocol) - Used to interact with HTTP and file-based NuGet feeds
-* [`NuGet.Packaging`](https://www.nuget.org/packages/NuGet.Packaging) - Used to interact with NuGet packages. `NuGet.Protocol` depends on this package
+* [`NuGet.CommandLine`](https://www.nuget.org/packages/NuGet.CommandLine) - NuGet Command Line Interface.
```

Comment:
This isn't part of the SDK. It's the package that contains an exe, and if you reference the package in a project, it will either fail restore, or it won't provide any APIs.